test-utils/runtime/src/system.rs

```diff
@@ -303,6 +305,20 @@ fn execute_storage_change(key: &[u8], value: Option<&[u8]>) -> ApplyExtrinsicRes
 	Ok(Ok(()))
 }
 
+fn execute_witness_backend() -> ApplyExtrinsicResult {
```

Comment:
What is the purpose of this test?

Reply:
It is just here to trigger wasm state machine no_std compilation error. I am not sure if there is an easier way to do it.

Reply:
Can we maybe add more? Like using some proof and check that it works?